packages/notification-managment-spa/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "notification-management-spa",
```

Comment:
As @sayak-sarkar said please change the name of spa and the folder file names associated with it

Reply:
Changed the folder name. That was missing